orbitdeterminator/kep_determination/lamberts_kalman.py

```diff
@@ -218,7 +218,12 @@ def kalman(kep, R):
         Pminus = 0.0  # a priori error estimate
         K = 0.0  # gain or blending factor
         # intial guesses
-        xhat = np.mean(kep[:, i])
+        try:
+            xhat = np.mean(kep[:, i])
+        except IndexError as err:
+            print('Error: {0} \n ** Switch Distance Units to metres **'.format(err))
```

Comment:
The error can be caused by something else too. Change the message to something like "Switching units to metres might help". Otherwise it's good.

Reply:
Sure. I'll make the changes.
Thanks for the review :D